checker/tests/nullness/java8inference/Issue402.java

```diff
@@ -3,13 +3,14 @@
 
 // @below-java8-jdk-skip-test
 
-// This test now seems to fail because of Issue 1006
-// @skip-test
-
 import java.util.Comparator;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
+@SuppressWarnings({
+    "nullness",
+    "keyfor"
+}) // Type argument inference does not infer the correct types.
```

Comment:
True.  Personall, I search for the key rather than the string `"@SuppressWarnings"`.  Maybe you do searches for `@SuppressWarnings`, so for you putting the warning on its line would be better.  There isn't a great solution here, unfortunately -- none of the three locations will accommodate all our needs.